README.md

```diff
@@ -1,12 +1,46 @@
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
-<!-- Copyright Contributors to the Egeria project. -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
 
-# Egeria
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3044/badge)](https://bestpractices.coreinfrastructure.org/projects/3044)
 
-<!-- TODO: Add a README -->
+[//]: # ([![Azure]&#40;https://dev.azure.com/odpi/egeria/_apis/build/status/odpi.egeria&#41;]&#40;https://dev.azure.com/odpi/Egeria/_build&#41;)
```

Comment:
azure is defunct effectively for us. As is the sonarcloud scan. Finally the maven central link should be updated to point to the artifact this repo is creating?

Reply:
seems to be a chicken-and-egg problem as we do not have an artifact yet.

Reply:
Fair point - but we know the name of the artifact.

Currently it is 
org.odpi.egeria:egeria-connector-integration-event-schema

It's just that it's only published as a snapshot.

I'd be inclined to update it anyway. Alternatively just remove completely and consider adding back after we've 'released' a first version (something we can do whenever is appropriate)